### ISSUE TITLE
Need Help

### ISSUE THREAD

**Issue by lihuanchen** [MiguelR0sa/NARX-Neural-Network-Hyper-Parameter-Tuning-And-Training--Matlab#1]
https://github.com/MiguelR0sa/NARX-Neural-Network-Hyper-Parameter-Tuning-And-Training--Matlab/blob/5559cde6612a007a0acff0c1272b03660dfe8cb8/HyperParameterTuning.m#L68

when I run this program,it tells me that 'nmseCell_M'is not recognized. How can I solve it?

**Comment by MiguelR0sa** [comment]
Hello,

If i remeber corretly, that is a function which takes the x and y inputs, plus a mask, all in cell format and computes the NMSE, which you must implement or replace/remove. The mask just comes from the data division done by the toolbox.

Good luck
________________________________
De: lihuanchen ***@***.***>
Enviado: 24 de maio de 2021 04:12
Para: MiguelR0sa/NARX-Neural-Network-Hyper-Parameter-Tuning-And-Training--Matlab ***@***.***>
Cc: Subscribed ***@***.***>
Assunto: [MiguelR0sa/NARX-Neural-Network-Hyper-Parameter-Tuning-And-Training--Matlab] Need Help (#1)


https://github.com/MiguelR0sa/NARX-Neural-Network-Hyper-Parameter-Tuning-And-Training--Matlab/blob/5559cde6612a007a0acff0c1272b03660dfe8cb8/HyperParameterTuning.m#L68

when I run this program,it tells me that 'nmseCell_M'is not recognized. How can I solve it?
